src/lighteval/tasks/extended/tiny_benchmarks/main.py

```diff
@@ -35,6 +35,7 @@
 from aenum import extend_enum
 from scipy.optimize import minimize
 
+import lighteval.tasks.tasks_prompt_formatting as prompt
```

Comment:
I think you forgot to push the `tasks_prompt_formatting` file

Reply:
Wdym?
This one? https://github.com/huggingface/lighteval/blob/main/src/lighteval/tasks/tasks_prompt_formatting.py

Reply:
Oh yeah I though you added this new file, but you just did not modify it my bad 😅